src/fswAlgorithms/attGuidance/attTrackingError/_UnitTest/test_attTrackingError.py

```diff
@@ -15,34 +15,17 @@
 # ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 # OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
-
-#
-#   Unit Test Script
-#   Module Name:        attTrackingError
-#   Author:             Hanspeter Schaub
-#   Creation Date:      January 15, 2016
-#
-
 import inspect
 import os
 
 import numpy as np
 
-# import packages as needed e.g. 'numpy', 'ctypes, 'math' etc.
-
 filename = inspect.getframeinfo(inspect.currentframe()).filename
```

Comment:
:small_red_triangle_down: These two lines and their associated imports can be deleted.